packages/experiments/src/common/TestImages.ts

```diff
@@ -1,5 +1,11 @@
 const baseProductionCdnUrl = 'https://az742526.vo.msecnd.net/files/odsp-next-release-odc_2018-04-13_20180418.001/odsp-media/images/apps/';
+const fabricAssetsCdnUrl = 'https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/';
```

Comment:
We have this all over @uifabric/experiments.. shame we don't export it from some common space... anyway that's orthogonal to this PR.

Reply:
Yep, seems like it's also in a couple config files. Will fix in a separate PR if i find some extra time ;)